templates/cases/case_read.haml

```diff
@@ -78,7 +78,7 @@
         .timeline{ ng-controller:"CaseTimelineController", ng-init:"init()" }
           .timeline-event.clearfix{ ng-repeat:"event in timeline | reverse", ng-class:'{ "timeline-action": (event.type == "A"), "timeline-incoming": (event.type == "I"), "timeline-outgoing": (event.type == "O") }' }
             .event-time
-              [[ event.time | autodate ]]
+              <cp-date-tooltip time=event.time position="top-right" />
```

Comment:
Wondering if we should wrap `event.time` in quotes, something like `time="{{ event.time }}"`. Not sure if haml does that automatically?